app/bundles/UserBundle/Controller/RoleController.php

```diff
@@ -279,7 +279,7 @@ public function editAction($objectId, $ignorePost = true)
 
         return $this->delegateView([
             'viewParameters' => [
-                'form'              => $form->createView(), //$this->setFormTheme($form, 'MauticUserBundle:Role:form.html.php', 'MauticUserBundle:FormTheme\Role'),
+                'form'              => $form->createView(),
```

Comment:
For these I would make sure the forms still render correctly. Are the themes being used?

Reply:
I believe this is handled by https://github.com/mautic/mautic/pull/12046. I'm lacking focus to the community in the past 2 weeks but don't want to block the merges.